### ISSUE TITLE
Should we use Hackerone to handle security reports?

### ISSUE THREAD

**Issue by Betree** [opencollective/opencollective#4815]
HackerOne is a very popular solution to handle security reports, the process of communicating with researchers and paying them. It has quite a large community, and projects are publicly listed there so that can bring a lot of attention.

**Pros**:
- Promotes to a large community of security researchers: more reports, better reports
- Public history: people will be able to see that we invest in security
- Standard tool with all the best practices to centralize reports
- [Free](https://www.hackerone.com/company/open-source-community) for open-source

**Cons**:
- Do we have the capacity to handle many reports (including very minor ones)?
- It's cool to eat our own dog food by using expenses to pay people

**Comment by SudharakaP** [comment]
I agree and love the idea, but reading more there might be one blocker in joining the community edition, which is the non-commercial requirement. The FAQ says,

>What are you looking for when approving an application?
>
>Our primary goal is to ensure that we are providing HackerOne's Community Edition for projects that are (a) genuinely Open Source, (b) are non-commercial, (c) will be able to run an effective security program, and (d) will utilize it as intended.

But still I guess we we can maybe apply and try it out. 🤔 😉

**Comment by Betree** [comment]
Seems https://huntr.dev could be super aligned with what we do:
- They are working solely with Open Source projects
- They are disbursing bounties through [Open Collective / OSC](https://opencollective.com/huntr) 
- They seem well integrated with Github
- They seem to have an active community

Could be nice to have a chat with them to see what we can do!
